section2/exercises/if-statements.js

```diff
@@ -83,14 +96,18 @@ When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pi
 */
 
 var cupsOfFlour = 1;
-var hasSauce = true;
+var hasSauce = false;
 
 // Write your conditional here
 
-
+if (cupsOfFlour >=2 && hasSauce === true) {
```

Comment:
VERY nice solution to this challenge; perfect use of the `&&` logical operator 💯